addons/escoria-core/game/core-scripts/log/esc_logger.gd

```diff
@@ -109,12 +109,7 @@ func warning(string: String, args = []):
 			_perform_stack_trace_log()
 			var message: String = ProjectSettings.get_setting(
 				"escoria/debug/crash_message"
-			) % [
-					log_file.get_path_absolute().get_base_dir().plus_file(
-						escoria.save_manager.crash_savegame_filename.get_file()
-					),
-					log_file.get_path_absolute()
-				]
+			) + "\n- " + log_file.get_path_absolute()
```

Comment:
I'd use a template string here instead of "+", but I'll accept the review regardless. You may decide if you change it.

Reply:
I didn't want to, because the message had 2 '%s', so templating won't work if I need only one file